### ISSUE TITLE
[Question] Is this the right tool for this job?

### ISSUE THREAD

**Issue by dancancro** [isteven/angular-multi-select#148]
Hi everyone,

I'd like to know whether this directive suits my requirements.

I want to make a page that is bound to a set of orders.  Each order has a set of one or more products stored as an array of ids.

I want to tell a directive 
1) what the complete set of products is, 
2) what the products of the specific order are
3) which field of the products objects should be stored in the order.products array (the id field)

I want to put it in my template like this
`<mydirective all-list="allProducts" selected-list="order.products" bound-field="id" ...>`

I don't want to have any extra code in my controller.  As soon as I read an order from the database and put it in $scope.order, the appropriate product checkboxes are checked.  As soon as I call update, the order.products field is updated in the database with an array matching what's checked.

In other words, I am looking for a multiselect directive that provides two way data-binding to an order with an array of products.

Does this do that?

Thanks,
Dan
